docs/src/examples/components/Icon/Usage/IconSetExample.shorthand.tsx

```diff
@@ -1,46 +1,46 @@
 import * as React from 'react'
-import { Grid, Divider, Header, Icon, Provider } from '@stardust-ui/react'
+import { Grid, Divider, Header, Icon, themes } from '@stardust-ui/react'
 
 const cellStyles = {
   margin: '10px 0',
 }
 
-const IconSetExampleShorthand = () => (
-  <Provider.Consumer
-    render={theme => (
-      <>
-        <div>
-          <Divider>
-            <Header as="h3" content="Regular" />
-          </Divider>
-          <Grid columns={4} styles={{ textAlign: 'center' }}>
-            {Object.keys(theme.icons).map(name => (
-              <div key={name} style={cellStyles}>
-                <Icon name={name} />
-                <br />
-                <code>{name}</code>
-              </div>
-            ))}
-          </Grid>
-        </div>
+const IconSetExampleShorthand: React.FunctionComponent<{ themeName: string }> = ({ themeName }) => (
+  <>
+    <div>
+      <Divider>
+        <Header as="h3" content="Regular" />
+      </Divider>
+      <Grid columns={4} styles={{ textAlign: 'center' }}>
+        {Object.keys(themes[themeName].icons).map(name => (
```

Comment:
I understand the changes here and the reason why they were introduced, but if client's is testing adding new icon on this example, these example won't show it. However, I don't have any better proposal for now :(

Reply:
> if client's is testing adding new icon on this example

Why? If it will be added directly to a theme it will work as expected...